src/locale/messages.xlf

```diff
@@ -564,6 +552,129 @@
           <context context-type="linenumber">5</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="0d4c8c660bb078b6ebb294da1f03d813885fe1d2" datatype="html">
+        <source>Cannot disable both IPv4 and IPv6</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">src/app/components/form/form.component.html</context>
+          <context context-type="linenumber">274,276</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="1899778155771790050" datatype="html">
+        <source>No data found for the zone.</source>
```

Comment:
Remove trailing dot

Reply:
I chose to add trailing dots to other notification message instead.